src/Assignment1/AssignmentService.java

```diff
@@ -0,0 +1,164 @@
+package Assignment1;
+
+public class AssignmentService {
+
+  private ItemModel model;
+
+  public AssignmentService() {}
+
+  public AssignmentService(ItemModel model) {this.model = model;}
+
+  // calculate tax methods
+  // method to calculate raw tax.
+  double rawTax(double total) {
+    if (total < 0) throw new RuntimeException("Total cant be -ve");
+    double tax = 0;
+    tax += total * 12.50 / 100;
+    return tax;
+  }
+
+  // method to calculate manufactured tax.
+  double manufacturedTax(double total) {
```

Comment:
Create separate Item class based on itemModel Type and use inheritance concept in it.